google-cloud-bigquery/acceptance/bigquery/query_destination_test.rb

```diff
@@ -63,7 +63,7 @@
   it "sends query results to a time partitioned destination table" do
     job = bigquery.query_job "SELECT * FROM UNNEST(" \
                              "GENERATE_TIMESTAMP_ARRAY(" \
-                             "'2018-10-01 00:00:00', '2018-10-10 00:00:00', " \
+                             "'2099-10-01 00:00:00', '2099-10-10 00:00:00', " \
```

Comment:
But what happens when these tests run in early fall in 2099?!?

Reply:
October 1 is my birthday; I am just setting a timer for some future maintainer to find this comment and wish me a happy birthday.